src/schema/v2/infiniteDiscovery/discoverArtworks.ts

```diff
@@ -54,6 +83,10 @@ export const DiscoverArtworks: GraphQLFieldConfig<void, ResolverContext> = {
       (node) => node.internalID
     )
 
+    if (sort == "certainty") {
+      artworkIds.reverse()
+    }
+
```

Comment:
**question (non-blocking):** What is the reasoning behind reversing the artwork IDs? Shouldn't the vector certainty (distance) already resolve the ordering?

Reply:
Good question! Weaviate returns items in descending order based of `certainty` score (i.e. artworks that are the most similar) will be returned first. But I want to experiment with returning artworks that are a not the most similar. So, for example, artworks that have a certainty score of `0.8` instead of `.98`. With this, I can set a certainty score of `0.8` and sort in ascending order, which will give me the artworks first.

The theory here is this might help with the issue of weaviate not providing enough variation in its recommendations.